<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="2" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Test Suite" duration-ms="0" started-at="2021-03-05T13:58:17 GMT" finished-at="2021-03-05T13:58:17 GMT">
    <groups>
    </groups>
  </suite> <!-- Test Suite -->
  <suite name="SuiteA" duration-ms="77694" started-at="2021-03-05T13:56:59 GMT" finished-at="2021-03-05T13:58:17 GMT">
    <groups>
    </groups>
    <test name="LoginTest" duration-ms="38425" started-at="2021-03-05T13:56:59 GMT" finished-at="2021-03-05T13:57:38 GMT">
      <class name="com.qtpselenium.hybrid.SuiteA.LoginTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="beforeTest" is-config="true" duration-ms="1579" started-at="2021-03-05T13:56:59 GMT" finished-at="2021-03-05T13:57:01 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="beforeMethod" is-config="true" duration-ms="183" started-at="2021-03-05T13:57:01 GMT" finished-at="2021-03-05T13:57:01 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="loginTest" duration-ms="34806" started-at="2021-03-05T13:57:01 GMT" data-provider="getData" finished-at="2021-03-05T13:57:36 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Success, RunMode=Y, Password=pass@1234, UserName=ashishthakur1983, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at com.qtpselenium.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:32)
at com.qtpselenium.hybrid.SuiteA.LoginTest.loginTest(LoginTest.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@id='userpass']/a"}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-7G1190LN', ip: '192.168.0.13', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 88.0.4324.27 (6347fe8bf1e48..., userDataDir: C:\Users\adi_d\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53990}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true, webdriver.remote.sessionid: de32c84263562dbab41574598fd...}
Session ID: de32c84263562dbab41574598fdc8db2
*** Element info: {Using=xpath, value=//input[@id='userpass']/a}
at org.testng.Assert.fail(Assert.java:99)
at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:114)
at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:92)
at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:73)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="afterMethod" is-config="true" duration-ms="144" started-at="2021-03-05T13:57:36 GMT" finished-at="2021-03-05T13:57:36 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="loginTest" duration-ms="1" started-at="2021-03-05T13:57:36 GMT" data-provider="getData" finished-at="2021-03-05T13:57:36 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Success, RunMode=N, Password=987654321a, UserName=987654321a, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[RunMode is set to N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: RunMode is set to N
at com.qtpselenium.hybrid.SuiteA.LoginTest.loginTest(LoginTest.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2021-03-05T13:57:36 GMT" finished-at="2021-03-05T13:57:36 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="afterMethod" is-config="true" duration-ms="0" started-at="2021-03-05T13:57:36 GMT" finished-at="2021-03-05T13:57:36 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.LoginTest@4e1d422d]" name="afterTest" is-config="true" duration-ms="1606" started-at="2021-03-05T13:57:36 GMT" finished-at="2021-03-05T13:57:38 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.qtpselenium.hybrid.SuiteA.LoginTest -->
    </test> <!-- LoginTest -->
    <test name="CreatePortfolioTest" duration-ms="77694" started-at="2021-03-05T13:56:59 GMT" finished-at="2021-03-05T13:58:17 GMT">
      <class name="com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="beforeTest" is-config="true" duration-ms="1571" started-at="2021-03-05T13:56:59 GMT" finished-at="2021-03-05T13:57:01 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="beforeMethod" is-config="true" duration-ms="183" started-at="2021-03-05T13:57:01 GMT" finished-at="2021-03-05T13:57:01 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="createPortfolioTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="createPortfolioTest" duration-ms="75384" started-at="2021-03-05T13:57:01 GMT" data-provider="getData" finished-at="2021-03-05T13:58:17 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, Password=pass@1234, PortfolioName=Patil123, UserName=ashishthakur1983, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at com.qtpselenium.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:32)
at com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest.createPortfolioTest(CreatePortfolioTest.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@id='userpass']/a"}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-7G1190LN', ip: '192.168.0.13', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 88.0.4324.27 (6347fe8bf1e48..., userDataDir: C:\Users\adi_d\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53989}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true, webdriver.remote.sessionid: ecb41177e03cf6871e554f20551...}
Session ID: ecb41177e03cf6871e554f20551bad17
*** Element info: {Using=xpath, value=//input[@id='userpass']/a}
at org.testng.Assert.fail(Assert.java:99)
at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:114)
at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:92)
at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:73)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPortfolioTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="afterMethod" is-config="true" duration-ms="369" started-at="2021-03-05T13:58:17 GMT" finished-at="2021-03-05T13:58:17 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2021-03-05T13:58:17 GMT" finished-at="2021-03-05T13:58:17 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="createPortfolioTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="createPortfolioTest" duration-ms="2" started-at="2021-03-05T13:58:17 GMT" data-provider="getData" finished-at="2021-03-05T13:58:17 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, Password=987654321a, PortfolioName=Patil123, UserName=987654321a, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[RunMode is set to N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: RunMode is set to N
at com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest.createPortfolioTest(CreatePortfolioTest.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPortfolioTest -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="afterTest" is-config="true" duration-ms="95" started-at="2021-03-05T13:58:17 GMT" finished-at="2021-03-05T13:58:17 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest@731a74c]" name="afterMethod" is-config="true" duration-ms="0" started-at="2021-03-05T13:58:17 GMT" finished-at="2021-03-05T13:58:17 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.qtpselenium.hybrid.SuiteA.CreatePortfolioTest -->
    </test> <!-- CreatePortfolioTest -->
  </suite> <!-- SuiteA -->
</testng-results>
